kubernetes/regapp/overlays/prod/kustomization.yml

```diff
@@ -17,7 +17,7 @@ resources:
 patchesStrategicMerge:
   - patches/regapp-db.yaml
   - patches/ingress.yaml
-  - patches/patch_authorization_url.yaml
```

Comment:
Looks like we can delete the `patches/patch_authorization_url.yaml` from prod overlay similar to what was done for dev in this PR unless I'm missing something?